### ISSUE TITLE
Library incompatible with node 16 and higher

### ISSUE THREAD

**Issue by pieterwillaert** [matteodelabre/saxophone#26]
```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'saxophone@0.7.2',
npm WARN EBADENGINE   required: { node: '10 || 12 || 14 || 15' },
npm WARN EBADENGINE   current: { node: 'v16.13.0', npm: '8.1.0' }
npm WARN EBADENGINE }
```

Node 16 is in Active LTS https://nodejs.org/en/about/releases/

**Comment by LaurensRietveld** [comment]
Note that adding `--ignore-engines` works: the library seems to compile and run fine. I.e., this seems like a package.json metadata issue

**Comment by wouterbeek** [comment]
For other lovers of old software out there, `yarn --ignore-engines` even works with the 2 year old Saxophone 0.6.1 version!

**Comment by DavideMontersino** [comment]
@wouterbeek - however in a CI/CD you will basically need to ignore all of the engines in every package, right? This workaround would not work. Also, not working for NPM afaik

**Comment by matteodelabre** [comment]
Fixed in release 0.8.0.
